setup/Microsoft.Build.swixproj

```diff
@@ -20,8 +20,8 @@
   <PropertyGroup>
     <OutDir>$(BaseOutputPath)setup\</OutDir>
     <OutputPath>$(BaseOutputPath)setup\</OutputPath>
-    <X86BinPath>$(BaseOutputPath)X86\$(OS)\$(Configuration)\</X86BinPath>
-    <X64BinPath>$(BaseOutputPath)X64\$(OS)\$(Configuration)\</X64BinPath>
+    <X86BinPath>$(BaseOutputPath)X86\$(OS)\$(Configuration)\Output\</X86BinPath>
+    <X64BinPath>$(BaseOutputPath)X64\$(OS)\$(Configuration)\Output\</X64BinPath>
```

Comment:
What's the motivation here? It seems fine but I don't understand why it's necessary.